### DISCUSSION TITLE
Production readiness for eBPF on windows

### DISCUSSION THREAD

**Discussion by absrivas12** [microsoft/ebpf-for-windows#3285] [Q&A] [answered]
The ebpf for windows project page positions the project as "work in progress".  Does this mean not recommended for production use?  Are there examples of production deployments?

**Comment by dahavey** [comment, marked as answer]
We have not published any MSFT signed binaries yet so you will have to use test-signing mode. This is tracked by #274. Could you please describe the use case for us to better help you?

**Comment by shyamvalsan** [comment]
@dahavey hello, I work at [Netdata](https://github.com/netdata/netdata) and we're very interested in this project. There's a lot of requests from the community for a native windows Netdata agent, currently we remotely monitor Windows servers using [windows exporter](https://github.com/prometheus-community/windows_exporter) but being able to collect metrics via eBPF directly, opens up a lot more possibilities. 

Do you have any rough roadmap or timelines, in terms of if/when this project will be publish signed binaries or be ready for production use?

**Reply by shpalani** [reply]
@shyamvalsan : Thanks for your interest in ebpf-for-windows platform.  
The response is stated in https://github.com/microsoft/ebpf-for-windows/discussions/3285#discussioncomment-8595553

While you wait for production-signed binaries, you can still do your PoC on test-signed binaries, if you have not already.

We have our weekly eBPF public office hours on Monday 8:30 PST time. You can join us and get all your questions answered. 
https://github.com/microsoft/ebpf-for-windows/discussions/427#discussion-3535657
